docs/core/tools/global-tools-how-to-use.md

```diff
@@ -28,6 +28,8 @@ This tutorial teaches you how to install and use a global tool. You use a tool t
 
    The `--add-source` parameter tells the .NET CLI to temporarily use the *./nupkg* directory as an additional source feed for NuGet packages. You gave your package a unique name to make sure that it will only be found in the *./nupkg* directory, not on the Nuget.org site.
 
+   [!INCLUDE[](~/includes/dotnet-tool-install-arch-options.md)]
```

Comment:
I would locate this after the command output to make it less prominent.

Reply:
Thanks Tom, moving these both to the location you suggested.